application/templates/mail/request_from_committed_to_committed.plush.html

```diff
@@ -0,0 +1,12 @@
+<h4><a href="<%= postURL %>"><%= postTitle %></a></h4>
```

Comment:
The name of this template doesn't match the content, but I assume that is what you're working on now.

Reply:
Yep.  Getting rid of that one and changing/adding others.